### ISSUE TITLE
Parse URL to gchange Ad

### ISSUE THREAD

**Issue by blavenie** [duniter-gchange/gchange-client#28]
In comment/description, parse URL to gchange web site : 
 - Ad
 - user profile

![image](https://user-images.githubusercontent.com/5269556/30371615-6d08ef98-987b-11e7-8c74-a67b89830c68.png)

**Comment by c-geek** [comment]
J'attendais cette réaction !

**Comment by blavenie** [comment]
oui, j'ai bien vu le clin d'oeil !  c'est comme le nom de profile "[cgeek] ..." : j'ai ca en tête de manière auto.
bref c'est une bonne manière de proposer les choses
